From c880396b5ddd5b15f5812801be15c428b801f599 Mon Sep 17 00:00:00 2001
From: Kazunori Kobayashi <kkobayas@igel.co.jp>
Date: Fri, 7 Jun 2013 15:44:11 +0900
Subject: [PATCH 02/21] h264parse: set GST_BUFFER_FLAG_HEADER flag if buffer
 contains SPS or PPS

This implementation notifies the next plugin of the buffer
containing SPS or PPS.
---
 gst/videoparsers/gsth264parse.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gst/videoparsers/gsth264parse.c b/gst/videoparsers/gsth264parse.c
index e57dcd0..59a6bfa 100644
--- a/gst/videoparsers/gsth264parse.c
+++ b/gst/videoparsers/gsth264parse.c
@@ -1441,6 +1441,7 @@ gst_h264_parse_push_codec_buffer (GstH264Parse * h264parse, GstBuffer * nal,
 
   GST_BUFFER_TIMESTAMP (nal) = ts;
   GST_BUFFER_DURATION (nal) = 0;
+  GST_BUFFER_FLAG_SET (nal, GST_BUFFER_FLAG_HEADER);
 
   return gst_pad_push (GST_BASE_PARSE_SRC_PAD (h264parse), nal);
 }
-- 
1.8.1.2

